### PR TITLE
WIP: Added additional datasets

### PR DESCRIPTION
**Pull request by kbgg**
There's still some fields which need to be filled out, they have TODO: located within them. Additionally, DOIs still need to be generated and added to the yaml files.

### CODE REVIEW

File: datasets/chesapeake_land_cover.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Name: Chesapeake Land Cover
+Description: "This dataset contains high-resolution aerial imagery from the USDA NAIP program, high-resolution land cover labels from the Chesapeake Conservancy, low-resolution land cover labels from the USGS NLCD 2011 dataset, low-resolution multi-spectral imagery from Landsat 8, and high-resolution building footprint masks from Microsoft Bing, formatted to accelerate machine learning research into land cover mapping. The Chesapeake Conservancy spent over 10 months and $1.3 million creating a consistent six-class land cover dataset covering the Chesapeake Bay watershed. While the purpose of the mapping effort by the Chesapeake Conservancy was to create land cover data to be used in conservation efforts, the same data can be used to train machine learning models that can be applied over even wider areas."
+Documentation: http://lila.science/datasets/chesapeakelandcover
+Contact: calebrob6+lcmcvpr2019@gmail.com
+CreatedBy: "[Labeled Information Library of Alexandria: Biology and Conservation](http://lila.science/)"
```
Comment (HamedAlemo) [inline datasets/chesapeake_land_cover.yaml:5]: replace with
[Chesapeake Conservancy](https://chesapeakeconservancy.org/), [Microsoft AI for Earth](https://www.microsoft.com/en-us/ai/ai-for-earth)

File: datasets/chesapeake_land_cover.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+Name: Chesapeake Land Cover
+Description: "This dataset contains high-resolution aerial imagery from the USDA NAIP program, high-resolution land cover labels from the Chesapeake Conservancy, low-resolution land cover labels from the USGS NLCD 2011 dataset, low-resolution multi-spectral imagery from Landsat 8, and high-resolution building footprint masks from Microsoft Bing, formatted to accelerate machine learning research into land cover mapping. The Chesapeake Conservancy spent over 10 months and $1.3 million creating a consistent six-class land cover dataset covering the Chesapeake Bay watershed. While the purpose of the mapping effort by the Chesapeake Conservancy was to create land cover data to be used in conservation efforts, the same data can be used to train machine learning models that can be applied over even wider areas."
+Documentation: http://lila.science/datasets/chesapeakelandcover
```
Comment (HamedAlemo) [inline datasets/chesapeake_land_cover.yaml:3]: Didn't Caleb send you a pdf documentation? If so add both links.

File: datasets/chesapeake_land_cover.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+Name: Chesapeake Land Cover
```
Comment (HamedAlemo) [inline datasets/chesapeake_land_cover.yaml:1]: Their paper's DOI that we can use: 10.1109/CVPR.2019.01301

File: datasets/chesapeake_land_cover.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Tags:
+  - building footprints
+  - land cover
+  - segmentation
+  - naip
+  - nlcd
+  - landsat 8
+Citation: "Robinson C, Hou L, Malkin K, Soobitsky R, Czawlytko J, Dilkina B, Jojic N. Large Scale High-Resolution Land Cover Mapping with Multi-Resolution Data. Proceedings of the 2019 Conference on Computer Vision and Pattern Recognition (CVPR 2019)."
+Resources:
+  - Description: NAIP 2011-2014 imagery
```
Comment (HamedAlemo) [inline datasets/chesapeake_land_cover.yaml:15]: All the licenses are incorrect. Some link to bigearthnet, landsat should not be Proprietary, and MSFT building footprints is ODBl. Please check them against the license section here: http://lila.science/datasets/chesapeakelandcover

File: datasets/cv4a_kenya_crop_challenge.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
+Name: CV4A Kenya Crop Type Competition
+Description: "TODO: Add description"
+Documentation: https://radiant-mlhub.s3-us-west-2.amazonaws.com/kenya-crop-challenge/FieldIds.csv
```
Comment (HamedAlemo) [inline datasets/cv4a_kenya_crop_challenge.yaml:3]: We had lost track of the documentation for this dataset. I added it to the gitlab repo (dev > Documentation.pdf). Please add that to the bucket and link it here.

File: datasets/cv4a_kenya_crop_challenge.yaml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+Contact: contact@radiant.earth
+CreatedBy: "[TODO: Add CreatedBy]()"
```
Comment (HamedAlemo) [inline datasets/cv4a_kenya_crop_challenge.yaml:5]: [Radiant Earth Foundation](https://www.radiant.earth/), [PlantVillage](https://plantvillage.psu.edu/)

File: datasets/cv4a_kenya_crop_challenge.yaml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+      AuthorURL: https://www.linkedin.com/in/hamedalemohammad/
+    - Title: A Guide to load the data in Python
+      URL: https://github.com/radiantearth/mlhub-tutorials/blob/master/notebooks/2020%20CV4A%20Crop%20Type%20Challenge/cv4a-crop-challenge-load-data.ipynb
+      AuthorName: Hamed Alemohammad
+      AuthorURL: https://www.linkedin.com/in/hamedalemohammad/
+    - Title: Visualizing Data in Python
+      URL: https://github.com/radiantearth/mlhub-tutorials/blob/master/notebooks/2020%20CV4A%20Crop%20Type%20Challenge/cv4a-crop-challenge-visualize-s2-data.ipynb
+      AuthorName: Marc Rußwurm
+      AuthorURL: https://www.linkedin.com/in/marcrusswurm/
+
```
Comment (HamedAlemo) [inline datasets/cv4a_kenya_crop_challenge.yaml:34]: add this repo:
https://github.com/sentinel-hub/cv4a-iclr-2020-starter-notebooks
authors is devisperessutti
Title: CV4A ICLR 2020 Starter Notebooks

File: datasets/cv4a_kenya_crop_challenge.yaml [hunk 8]
```diff
@@ -0,0 +1,36 @@
+Name: CV4A Kenya Crop Type Competition
+Description: "TODO: Add description"
+Documentation: https://radiant-mlhub.s3-us-west-2.amazonaws.com/kenya-crop-challenge/FieldIds.csv
+Contact: contact@radiant.earth
```
Comment (HamedAlemo) [inline datasets/cv4a_kenya_crop_challenge.yaml:4]: ml@radiant.earth

File: datasets/dalberg_data_insights_crop_type_uganda.yaml [hunk 9]
```diff
@@ -0,0 +1,28 @@
+Name: Dalberg Data Insights Crop Type Uganda
+Description: "TODO: Add description"
+Documentation: https://radiant-mlhub.s3-us-west-2.amazonaws.com/african-crops-01/Uganda_Documentation.pdf
+Contact: contact@radiant.earth
```
Comment (HamedAlemo) [inline datasets/dalberg_data_insights_crop_type_uganda.yaml:4]: Christophe Bocquet <Christophe.Bocquet@dalberg.com>

File: datasets/dalberg_data_insights_crop_type_uganda.yaml [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+CreatedBy: "[TODO: Add CreatedBy]()"
```
Comment (HamedAlemo) [inline datasets/dalberg_data_insights_crop_type_uganda.yaml:5]: [Dalberg Data Insights](https://dalberg.com/what-we-do/dalberg-data-insights/), [Radiant Earth Foundation](https://www.radiant.earth/)

File: datasets/dalberg_data_insights_crop_type_uganda.yaml [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+Tags:
+  - crop type
+  - segmentation
+  - sentinel-2
+Citation: "Bocquet, C., Dalberg Data Insights. (2019) Dalberg Data Insights Uganda Crop Classification, Version 1. [Indicate subset used]. Radiant ML Hub. [Date Accessed]"
+Resources:
```
Comment (HamedAlemo) [inline datasets/dalberg_data_insights_crop_type_uganda.yaml:11]: Both of the resources here should be CC-BY-4.0.

File: datasets/great_african_food_company_crop_type_tanzania.yaml [hunk 12]
```diff
@@ -0,0 +1,28 @@
+Name: Great African Food Company Crop Type Tanzania
+Description: "TODO: Add description"
+Documentation: https://radiant-mlhub.s3-us-west-2.amazonaws.com/african-crops-01/Tanzania_Documentation.pdf
+Contact: contact@radiant.earth
+CreatedBy: "[TODO: Add CreatedBy]()"
```
Comment (HamedAlemo) [inline datasets/great_african_food_company_crop_type_tanzania.yaml:5]: [Great African Food Company](https://www.linkedin.com/company/gafco/), [Radiant Earth Foundation](https://www.radiant.earth/)

File: datasets/great_african_food_company_crop_type_tanzania.yaml [hunk 13]
```diff
@@ -0,0 +1,28 @@
+Name: Great African Food Company Crop Type Tanzania
+Description: "TODO: Add description"
+Documentation: https://radiant-mlhub.s3-us-west-2.amazonaws.com/african-crops-01/Tanzania_Documentation.pdf
+Contact: contact@radiant.earth
```
Comment (HamedAlemo) [inline datasets/great_african_food_company_crop_type_tanzania.yaml:4]: ml@radiant.earth

File: datasets/great_african_food_company_crop_type_tanzania.yaml [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+CreatedBy: "[TODO: Add CreatedBy]()"
+Tags:
+  - crop type
+  - segmentation
+  - sentinel-2
+Citation: "Great African Food Company. (2019) Great African Food Company Tanzania Ground Reference Crop Type Dataset, Version 1. [Indicate subset used]. Radiant ML Hub. [Date Accessed]"
+Resources:
```
Comment (HamedAlemo) [inline datasets/great_african_food_company_crop_type_tanzania.yaml:11]: Similarly change licenses to CC-BY-4.0

File: datasets/plantvillage_crop_type_kenya.yaml [hunk 15]
```diff
@@ -0,0 +1,28 @@
+Name: PlantVillage Crop Type Kenya
+Description: "TODO: Add description"
+Documentation: https://radiant-mlhub.s3-us-west-2.amazonaws.com/african-crops-01/Kenya_Documentation.pdf
+Contact: contact@radiant.earth
```
Comment (HamedAlemo) [inline datasets/plantvillage_crop_type_kenya.yaml:4]: ml@radiant.earth

File: datasets/plantvillage_crop_type_kenya.yaml [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+CreatedBy: "[TODO: Add CreatedBy]()"
```
Comment (HamedAlemo) [inline datasets/plantvillage_crop_type_kenya.yaml:5]: [PlantVillage](https://plantvillage.psu.edu/), [Radiant Earth Foundation](https://www.radiant.earth/)

File: datasets/spacenet_2.yaml [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+Today, map features such as roads, building footprints, and points of interest are primarily created through manual techniques. We believe that advancing automated feature extraction techniques will serve important downstream uses of map data including humanitarian and disaster response, as observed by the need to map road networks during the response to recent flooding in Bangladesh and Hurricane Maria in Puerto Rico. Furthermore, we think that solving this challenge is an important stepping stone to unleashing the power of advanced computer vision algorithms applied to a variety of remote sensing data applications in both the public and private sector."
+Documentation: https://spacenetchallenge.github.io/datasets/spacenetBuildings-V2summary.html
+Contact: https://spacenet.ai/contact-us/
+CreatedBy: "[SpaceNet LLC](https://spacenet.ai/)"
+Tags:
+  - building footprints
+  - segmentation
+  - worldview-3
+Citation: "SpaceNet on Amazon Web Services (AWS). “Datasets.” The SpaceNet Catalog.  Last modified April 30, 2018. Accessed on [Insert Date]. https://spacenetchallenge.github.io/datasets/datasetHomePage.html."
+Resources:
```
Comment (HamedAlemo) [inline datasets/spacenet_2.yaml:13]: Q: They also have test dataset. To confirm, we haven't added the test datasets right?

File: datasets/spacenet_2.yaml [hunk 18]
```diff
@@ -0,0 +1,33 @@
...
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+  - Description: SpaceNet 2 Shanghai Chipped Training Dataset
+    CollectionID: sn_AOI_4_Shanghai
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+  - Description: SpaceNet 2 Khartoum Chipped Training Dataset
+    CollectionID: sn_AOI_5_Khartoum
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+DataAtWork:
```
Comment (HamedAlemo) [inline datasets/spacenet_2.yaml:30]: Publication:
https://medium.com/the-downlinq/2nd-spacenet-competition-winners-code-release-c7473eea7c11

Tools:
https://github.com/SpaceNetChallenge/BuildingDetectors_Round2

File: datasets/chesapeake_land_cover.yaml (hunk 2 above)
Comment (kbgg) [reply datasets/chesapeake_land_cover.yaml:3]: No, Caleb was going to just send over their website converted to a PDF so we decided on linking to the website instead as that will always be up to date.

File: datasets/spacenet_2.yaml (hunk 17 above)
Comment (kbgg) [reply datasets/spacenet_2.yaml:13]: This is for their spacenet 2 dataset, which they had generated the catalog for. I don't think the differentiate between test and training data in their catalog.

Comment (HamedAlemo) [reply datasets/spacenet_2.yaml:13]: They do actually, see bottom of this page. They have training data and test data.

Comment (kbgg) [reply datasets/spacenet_2.yaml:13]: Correct, but this STAC catalog was generated by Nick. In the script they used they only include training data in the catalog

Comment (HamedAlemo) [reply datasets/spacenet_2.yaml:13]: Great, just wanted to make sure we use the right terminology in the description.

File: datasets/cv4a_kenya_crop_challenge.yaml [hunk 19]
```diff
@@ -0,0 +1,46 @@
...
+<br><br>
+Each tile has a) 13 multi-band observations throughout the growing season. Each observation includes 12 bands from Sentinel-2 L2A product, and a cloud probability layer. The twelve bands are [B01, B02, B03, B04, B05, B06, B07, B08, B8A, B09, B11, B12]. The cloud probability layer is a product of the Sentinel-2 atmospheric correction algorithm (Sen2Cor) and provides an estimated cloud probability (0-100%) per pixel. All of the bands are mapped to a common 10 m spatial resolution grid.; b) A raster layer indicating the crop ID for the fields in the training set; and c) A raster layer indicating field IDs for the fields (both training and test sets). Fields with a crop ID of 0 are the test fields."
+Documentation: https://radiant-mlhub.s3-us-west-2.amazonaws.com/kenya-crop-challenge/Documentation.pdf
+Contact: ml@radiant.earth
+CreatedBy: "[Radiant Earth Foundation](https://www.radiant.earth/), [PlantVillage](https://plantvillage.psu.edu/)"
+Tags:
+  - crop type
+  - segmentation
+  - sentinel-2
+Citation: "Radiant Earth Foundation (2020) CV4A Competition Kenya Crop Type Dataset, Version 1. [Indicate subset used]. Radiant ML Hub. [Date Accessed]"
```
Comment (HamedAlemo) [inline datasets/cv4a_kenya_crop_challenge.yaml:16]: DOI: 10.34911/rdnt.dw605x

File: datasets/dalberg_data_insights_crop_type_uganda.yaml [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+<br><br>
+Post ground data collection, Radiant Earth Foundation conducted a quality control of the polygons using Sentinel-2 imagery of the growing season as well as Google basemap imagery, and removed several polygons that overlapped with infrastructure or built-up areas. Finally, ground reference polygons were matched with corresponding time series data from Sentinel-2 satellites (listed in the source imagery property of each label item)."
+Documentation: https://radiant-mlhub.s3-us-west-2.amazonaws.com/african-crops-01/Uganda_Documentation.pdf
+Contact: Christophe.Bocquet@dalberg.com
+CreatedBy: "[Dalberg Data Insights](https://dalberg.com/what-we-do/dalberg-data-insights/), [Radiant Earth Foundation](https://www.radiant.earth/)"
+Tags:
+  - crop type
+  - segmentation
+  - sentinel-2
+Citation: "Bocquet, C., Dalberg Data Insights. (2019) Dalberg Data Insights Uganda Crop Classification, Version 1. [Indicate subset used]. Radiant ML Hub. [Date Accessed]"
```
Comment (HamedAlemo) [inline datasets/dalberg_data_insights_crop_type_uganda.yaml:12]: DOI: 10.34911/rdnt.eii04x

File: datasets/great_african_food_company_crop_type_tanzania.yaml [hunk 21]
```diff
@@ -0,0 +1,30 @@
...
+Documentation: https://radiant-mlhub.s3-us-west-2.amazonaws.com/african-crops-01/Tanzania_Documentation.pdf
+Contact: ml@radiant.earth
+CreatedBy: "[Great African Food Company](https://www.linkedin.com/company/gafco/), [Radiant Earth Foundation](https://www.radiant.earth/)"
+Tags:
+  - crop type
+  - segmentation
+  - sentinel-2
+Citation: "Great African Food Company. (2019) Great African Food Company Tanzania Ground Reference Crop Type Dataset, Version 1. [Indicate subset used]. Radiant ML Hub. [Date Accessed]"
```
Comment (HamedAlemo) [inline datasets/great_african_food_company_crop_type_tanzania.yaml:12]: DOI: 10.34911/rdnt.5vx40r

File: datasets/plantvillage_crop_type_kenya.yaml [hunk 22]
```diff
@@ -0,0 +1,30 @@
...
+<br><br>
+Post ground data collection, Radiant Earth Foundation conducted a quality control of the polygons using Sentinel-2 imagery of the growing season as well as Google basemap imagery. Two actions were taken on the data 1)several polygons that had overlapping areas with different crop labels were removed, 2) invalid polygons where multiple points were collected in corners of the field (within a distance of less than 0.5m) and the overall shape was not convex, were corrected. Finally, ground reference polygons were matched with corresponding time series data from Sentinel-2 satellites (listed in the source imagery property of each label item). "
+Documentation: https://radiant-mlhub.s3-us-west-2.amazonaws.com/african-crops-01/Kenya_Documentation.pdf
+Contact: ml@radiant.earth
+CreatedBy: "[PlantVillage](https://plantvillage.psu.edu/), [Radiant Earth Foundation](https://www.radiant.earth/)"
+Tags:
+  - crop type
+  - segmentation
+  - sentinel-2
+Citation: "PlantVillage. (2019) PlantVillage Kenya Ground Reference Crop Type Dataset, Version 1. [Indicate subset used]. Radiant ML Hub. [Date Accessed]"
```
Comment (HamedAlemo) [inline datasets/plantvillage_crop_type_kenya.yaml:12]: DOI: 10.34911/rdnt.u41j87

**Pull request merged**